make.sh

```diff
@@ -3,21 +3,42 @@
 # Unicorn Engine
 # By Nguyen Anh Quynh <aquynh@gmail.com>, 2015
 
-MAKE_JOBS=$((${MAKE_JOBS}+0))
+usage() {
+    cat 1>&2 <<EOF
+make.sh - The build script for unicorn engine
+USAGE:
+    $ ./make.sh [OPTIONS]
+OPTIONS:
+                            Build the project
+    asan                    Build for ASan
+    install                 Install the project
+    uninstall               Uninstall the project
+    macos-universal         Build universal binaries on macOS
+    macos-universal-no      Build non-universal binaries that includes only 64-bit code on macOS
+    cross-win32             Cross-compile Windows 32-bit binary with MinGW
+    cross-win64             Cross-compile Windows 64-bit binary with MinGW
+    cross-android           Cross-compile for Android
+    ios                     Cross-compile for all iDevices (armv7 + armv7s + arm64)
+    ios_armv7               Cross-compile for ArmV7 (iPod 4, iPad 1/2/3, iPhone4, iPhone4S)
+    ios_armv7s              Cross-compile for ArmV7s (iPad 4, iPhone 5C, iPad mini)
+    ios_arm64               Cross-compile for Arm64 (iPhone 5S, iPad mini Retina, iPad Air)
+    linux32                 Cross-compile Unicorn on 64-bit Linux to target 32-bit binary
+EOF
+}
+
+MAKE_JOBS=$((MAKE_JOBS+0))
 [ ${MAKE_JOBS} -lt 1 ] && \
   MAKE_JOBS=4
 
 # build for ASAN
 asan() {
-  UNICORN_DEBUG=yes
-  UNICORN_ASAN=yes
-  ${MAKE} V=1
+  env UNICORN_DEBUG=yes UNICORN_ASAN=yes "${MAKE}" V=1
 }
 
 # build iOS lib for all iDevices, or only specific device
 build_iOS() {
-  IOS_SDK=`xcrun --sdk iphoneos --show-sdk-path`
-  IOS_CC=`xcrun --sdk iphoneos -f clang`
+  IOS_SDK=$(xcrun --sdk iphoneos --show-sdk-path)
```

Comment:
@radare what do you think, i am unsure if this is a good fix, that is portable across all shells?

Reply:
I think it is portable. http://pubs.opengroup.org/onlinepubs/009695399/utilities/xcu_chap02.html#tag_02_06_03

Reply:
bash? but we do not depend on a specific shell, so please make sure this works with other non-bash shells too.

Reply:
I think all POSIX shell should follow [the opengroup](https://en.wikipedia.org/wiki/The_Open_Group) specification above.